src/Graph5/Graph5.tsx

```diff
@@ -0,0 +1,46 @@
+import { AreaChart, Area, ResponsiveContainer } from "recharts";
+import data from "./DataGraph5.json";
+
+function findHighestUV(data: { name: string; uv: number }[]): number {
+	const uvValues = data.map((item) => item.uv);
+	const highestUV = Math.max(...uvValues);
+	return highestUV;
+}
+
+const highestUVValue = findHighestUV(data);
+
+export function Graph5() {
+	return (
+		<div>
+			<h1 className="text-2xl  font-bold">{highestUVValue}</h1>
+			<span className="bg-teal-400 h-2 w-2 rounded-full inline-block"></span>
+			<ResponsiveContainer width="100%" height={150}>
+				<AreaChart
+					width={200}
+					height={60}
+					data={data}
+					margin={{
+						top: 5,
+						right: 0,
+						left: 0,
+						bottom: 5,
+					}}
+				>
+					<defs>
+						<linearGradient id="colorUv" x1="0" y1="0" x2="0" y2="1">
+							<stop offset="5%" stopColor="#0772d7" stopOpacity={0.8} />
+							<stop offset="95%" stopColor="#0772d7" stopOpacity={0} />
+						</linearGradient>
+					</defs>
+					<Area
```

Comment:
Dodaj moze strokeWidth na 2 albo 3, wydaje mi sie ze lepiej troche wyglada wtedy

Reply:
I found it here https://recharts.org/en-US/api/Area
Other methods:
1. use option + esc on macos to display available props for a component in vscode 
![image](https://github.com/Zorkann/future-dashboard/assets/28710560/c86c65da-6a59-42bd-8365-9c4d936543ad)
2. Click on the component with pressed command key in order to go to types definition and then you can check it here:
![image](https://github.com/Zorkann/future-dashboard/assets/28710560/78ef368c-da0b-49ed-ae50-734d9a962ecd)

Reply:
done, thanks for sugesstions